content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -132,6 +132,9 @@ scheduler will call each scoring plugin for each node. There will be a well
 defined range of integers representing the minimum and maximum scores. After the
 [NormalizeScore](#normalize-scoring) phase, the scheduler will combine node
 scores from all plugins according to the configured plugin weights.
+With the enabling of the `StorageCapacityScoring` feature gate, nodes can be
+scored based on their available storage capacity. This only applies when using
+no static provisioning (dynamic provisioning only).
```

Comment:
Try something like this:
```suggestion

#### Capacity scoring {#scoring-capacity}

{{< feature-state feature_gate_name="StorageCapacityScoring" >}}

As an alpha feature, nodes can be scored based on their available storage capacity.
This only applies when not using static provisioning
([dynamic provisioning](/docs/concepts/storage/dynamic-provisioning/) only).
```

See https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates for more details about the extra metadata you need to create so that this will work.

Reply:
Thank you for your suggestion. I am currently implementing the feature, and I will address it as soon as it is completed.